src/Wordpress/CustomPostType/Booking.php

```diff
@@ -31,21 +33,6 @@ public function __construct() {
 		// does not trigger when initiated in initHooks
 		add_action( 'post_updated', array( $this, 'postUpdated' ), 1, 3 );
 
-		// Frontend request
-		try {
-			$this->handleFormRequest();
```

Comment:
Das wurde vorher über den Konstruktor von dem CPT aufgerufen. Dieser Aufruf liegt jetzt separat in der Plugin.php. Das musste ich so umstellen, weil ich für die Buchungsregeln schon beim Buchungsprozess auf die Taxonomien zugreifen muss (um zu schauen ob diese zutreffen).

Reply:
Genau, vorher wurde über den init Post Type die registerCustomPostTypes aufgerufen die dann über Umwege den Konstruktor hier aufgerufen hat. Dieser Konstruktor wieder hat dann die handleFormRequest aufgerufen. Das habe ich jetzt da dann direkt im Hook gemacht,